data/scenario-test.cfg

```diff
@@ -4380,6 +4397,34 @@ unplagueable: $wml_unit.status.unplagueable"
             advance=false
         [/unstore_unit]
     [/event]
+    {FORCE_CHANCE_TO_HIT (id=cth_tester) () 0 ()}
+    [event]
+        name=attacker hits
+        first_time_only=yes
+        [filter]
+            id=cth_tester
+        [/filter]
+
+        [message]
+            speaker=narrator
+            image=wesnoth-icon.png
+            message="FORCE_CHANCE_TO_HIT macro don't working fine. "
```

Comment:
If this test is going to exist, we need to make the messages actually comprehensible.